src/UI/Actions.hs

```diff
@@ -759,6 +789,24 @@ pipeToCommand =
     in Brick.suspendAndResume $ liftIO $ pipeCommand' s cmd
   }
 
+-- | Save a currently selected attachment to the given path on
+-- disk. Will error if the path is invalid or can not be written
+-- to. Otherwise, write out the attachment to a temporary file and then
+-- rename it to the path given by the user.
```

Comment:
The doc says you write to temp file then rename.  But this doesn't look like what is happening in `writeEntityToPath`.  Did I miss something, or is the doc wrong?

Reply:
Oof why don't I see those things.

I had it implemented first with that in mind, but then decided that it was overkill and got rid of it. I'll adjust the doc string.